.github/workflows/ci.yml

```diff
@@ -218,6 +218,14 @@ jobs:
           regex: '^refs/tags/v(((([0-9]+)\.[0-9]+)\.[0-9]+)(-.+)?)$'
         if: ${{ startsWith(github.ref, 'refs/tags/v') }}
 
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin' # Temurin is cached on GitHub Actions Runners.
+          java-version: '17'
+          cache: 'gradle'
+        if: ${{ contains('apk appbundle', matrix.platform) }}
```

Comment:
We have to bump up the Java version used from 8 to 17 due to the updated `package_info_plus` package using it.

GitHub Runners have 8, 11, 17 and 21 versions installed on Ubuntu runners, yet by default 11 is used. So this step, when using Eclipse Temurin distribution, [that is cached on runners](https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#hosted-tool-cache), takes 1-2 seconds to complete only, as it only sets up the JAVA_HOME env to be 17 version.

For local development, Android Studio already bundles Java version 17 and higher, starting from 2021 year release.